NextcloudTalk/WSMessage.m

```diff
@@ -134,10 +129,11 @@ - (void)sendMessageWithWebSocket:(NSURLSessionWebSocketTask *)webSocketTask
     }
 
     NSLog(@"Sending: %@", self.webSocketMessage);
+    [NCUtils trace:[NSString stringWithFormat:@"Sending: %@", self.webSocketMessage]];
```

Comment:
Do you want to add trace() to this PR or was it just for testing purposes? :)

Reply:
Oops, I added it locally, but wasn't meant for the PR 😅